### PR TITLE
password_verify incorrect use

### PR DESCRIPTION
**Pull request by chaegumi**

### CODE REVIEW

File: examples/src/Repositories/ClientRepository.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function getClientEntity($clientIdentifier, $grantType, $clientSecret = n
     {
         $clients = [
             'myawesomeapp' => [
+				'passwd'          => 'abc123',
```
Comment (lookyman) [inline examples/src/Repositories/ClientRepository.php]: You have tabs here instead of spaces.

**Comment by alexbilbie** [comment]
Your changes aren't correct I'm afraid.

The `$clientSecret` is the plain text that wants verifying against the hash

**Comment by chaegumi** [comment]
http://www.php.net/manual/en/function.password-verify.php

·boolean password_verify ( string $password , string $hash )·

```php

<?php
// See the password_hash() example to see where this came from.
$hash = '$2y$07$BCryptRequires22Chrcte/VlQH0piJtjXl.0t1XkA8pw9dMXTpOq';

if (password_verify('rasmuslerdorf', $hash)) {
    echo 'Password is valid!';
} else {
    echo 'Invalid password.';
}
?>

```

**Comment by alexbilbie** [comment]
Where is `$clientSecret` a hashed value?

**Comment by chaegumi** [comment]
Oh, It's my fault, I passed the hashed value.

```php

'form_params' => [
               'grant_type' => 'authorization_code',
               'client_id' => 'myawesomeapp',
               'client_secret' => '$2y$10$0cnZgLgUlIBkges1dDxsYe4w9fSvrj7zpeg33Lj7QdDbG6k4tYZOW', // Here must be abc123 
               'redirect_uri' => 'http://foo/callback',
               'code' => $code
           ]

```

The correct use is:

```php

'form_params' => [
               'grant_type' => 'authorization_code',
               'client_id' => 'myawesomeapp',
               'client_secret' => 'abc123', 
               'redirect_uri' => 'http://foo/callback',
               'code' => $code
           ]

```

**Pull request closed without merging**